simplesource-command-kafka/src/main/java/io/simplesource/kafka/internal/streams/topologies/EventSourcedTopologyBuilder.java

```diff
@@ -0,0 +1,67 @@
+package io.simplesource.kafka.internal.streams.topologies;
```

Comment:
All sub-topology classes are initialised with `AggregateTopologyContext` instead of passing only required dependencies to each sub-topology class. So, no need to have `EventSourcedTopologyBuilder` anymore